ariadne/asgi/handlers/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .graphql_transport_ws import GraphQLTransportWS
+from .graphql_ws import GraphQLWS
```

Comment:
I don't think we need to prefix all modules with `graphql`.

Handlers should have `Handler` suffix, eg. `GraphQLHTTPHandler`.

Reply:
Module names are meant to reflect the protocol names, as the protocols are: "graphql-transport-ws" and "graphql-ws". Does it work for you?